src/lib/holocene/input/input.svelte

```diff
@@ -47,7 +51,19 @@
         <Icon name={$copied ? 'checkMark' : 'copy'} stroke="currentcolor" />
       </div>
     {/if}
+    {#if maxLength}
+      <span class="count">
+        <span
+          class="text-blue-700"
+          class:warn={maxLength - value.length <= 5}
+          class:error={maxLength === value.length}>{value.length}</span
+        >&nbsp;/&nbsp;{maxLength}
+      </span>
+    {/if}
   </div>
+  {#if hintText}
+    <span class="mt-1 text-xs text-red-700">{hintText}</span>
```

Comment:
We might want to consider accessibility for this hint in the future (e.g. adding an `aria-live` attribute).